collections/src/main/scala/strawman/collection/immutable/ImmutableArray.scala

```diff
@@ -197,6 +199,10 @@ object ImmutableArray extends StrictOptimizedSeqFactory[ImmutableArray] {
       case that: ofByte => Arrays.equals(unsafeArray, that.unsafeArray)
       case _ => super.equals(that)
     }
+    override def slice(from: Int, until: Int): ImmutableArray[Byte] = {
+      val lo = scala.math.max(from, 0)
+      ImmutableArray.unsafeWrapArray(Arrays.copyOfRange(unsafeArray, lo, until))
```

Comment:
This isn't correct.  `copyOfRange` pads with zeros rather than truncating when you run off the end.

Reply:
@Ichoran fixed. Thank you.